README.md

```diff
@@ -1257,7 +1257,7 @@ You can use Gitpod, an online IDE(which is free for Open Source) for contributin
 
 ## Resources
 
-* [Changelog](https://github.com/axios/axios/blob/master/CHANGELOG.md)
+* [Changelog](https://github.com/axios/CHANGELOG.md)
```

Comment:
Please can you change all of them in this PR 😃

Reply:
@jasonsaayman 

Is this better?